home_work_3.rb

```diff
@@ -0,0 +1,20 @@
+class Item
+  
+    attr_reader :whore, :prise, :weight, :breast_size
+    attr_writer :whore, :prise, :weight, :breast_size
+    
+    def initialize(whore, prise, weight, breast_size)
+      @whore = whore
+      @prise = prise
+      @weight = weight
+      @breast_size = breast_size
+    end
+  
+  end  
+  
+  item1 = Item.new("Margo", "100$", "55", "3")
```

Comment:
```suggestion
  item1 = Item.new("Margo", 100.0, 55.0)
```
аргументы для веса и цены у нас флоат. Все как в реальной жизни. никаких строк